webapp/src/main/java/com/box/l10n/mojito/entity/ThirdPartyFileChecksum.java

```diff
@@ -0,0 +1,69 @@
+package com.box.l10n.mojito.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.Index;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import org.joda.time.DateTime;
+
+/** Entity that stores the checksum of a translated file downloaded via a third party sync. */
+@Entity
+@Table(
+    name = "third_party_sync_file_checksum",
+    indexes = {
+      @Index(
+          name = "I__THIRD_PARTY__CHECKSUM__REPOSITORY_ID__FILE_NAME__LOCALE_ID",
```

Comment:
the index name is `I__{table_name}__{column1_name}__{column2_name}...` so i think here it should be something like `I__TP_FILE_CHECKSUM__REPOSITORY_ID__LOCALE_ID__FILE_NAME` eventually shortening some part